static/js/compose.js

```diff
@@ -601,7 +573,7 @@ function validate_stream_message_post_policy(sub) {
     }
 
     if (stream_post_policy === stream_post_permission_type.moderators.code) {
-        compose_error(
+        compose_error.compose_error(
```

Comment:
What if we call this `compose_error.show()`?  `compose_error.compose_error` feels awkward to read.